skimage/measure/tests/test_regionprops.py

```diff
@@ -1550,3 +1559,31 @@ def test_pickling_region_properties():
     pickled = pickle.dumps(regions[0])
     unpickled = pickle.loads(pickled)  # RecursionError here
     assert regions[0] == unpickled
+
+
+@pytest.mark.parametrize("old_name", list(PROPS))
+def test_deprecated_properties(old_name):
+    regions = regionprops(SAMPLE, intensity_image=INTENSITY_SAMPLE)
+
+    regex = rf"`RegionProperties\['{old_name}'\]` is deprecated"
+    with pytest.warns(FutureWarning, match=regex) as record:
+        result = regions[0][old_name]
+    testing.assert_stacklevel(record)
+
+    current_name = PROPS[old_name]
+    np.testing.assert_equal(result, regions[0][current_name])
+
+    if old_name == old_name.lower():
+        # lower case names in PROPS are still accessible as attributes
+        # Make sure those emit an appropriate warning
```

Comment:
To do: clarify this comment.

Reply:
@stefanv, let me know if https://github.com/scikit-image/scikit-image/pull/7778/commits/cabaf1a5d741421c0549f4e8daff7305a390d0d6 resolves this. 

Don't really know why only lower-case properties were made accessible as attributes. Maybe that was introduced after the switch from camel case to snake case.